README.md

```diff
@@ -266,159 +236,37 @@ F1_mean.attach(engine, "F1")
 
 # Installation
 
-From [pip](https://pypi.org/project/pytorch-ignite/):
-
-```bash
-pip install pytorch-ignite
-```
-
-From [conda](https://anaconda.org/pytorch/ignite):
-
-```bash
-conda install ignite -c pytorch
-```
```

Comment:
I'd like to keep these 2 installation lines for `pip` and `conda`, so no need to jump into another website, in most cases you just need to copy-paste a command